kimchi/wasm/src/plonk_verifier_index.rs

```diff
@@ -20,6 +20,41 @@ use std::path::Path;
 use std::sync::Arc;
 use wasm_bindgen::prelude::*;
 
+pub struct SimpleFeatureFlags {
+    pub range_check0: bool,
+    pub range_check1: bool,
```

Comment:
Just a clarifying question for me: what does `range_check1` correspond to? `range_check0` is the `64` bit range check; what is the other one?

Reply:
Here's how I understand it (@jspada would know the full details):

`range_check0` actually has space for up to 88 bits (for the 64-bit check we set the upper two 12-bit limbs to constant zero). However, when used in the 88-bit mode it doesn't fully constrain all limbs by itself and needs to "collaborate" with subsequent gates.

The main objective for both range check gates was to enable a 3x88-bit check (used in foreign field arithmetic, supports foreign field element sizes up to 264 = 3 * 88 bits). The `range_check1` gate is what is needed together with two `range_check0` gates and one `Zero` gate to collaboratively do this 3*88 bit check.

Note that that's 264 bits of range check in 4 gates, which is 264 / 4 = 66 bits per gate on average. So the collaborative approach gives you a 2 bit edge over the single gate version here 😅 Just doing 4 64-bit range checks separately would have only given us 265 bit checks which would've been to small for important use cases in foreign field multiplication.

Reply:
check out https://github.com/MinaProtocol/mina/blob/e2b96abe89faf0caa92a75495b9b4b0cb96f2d58/src/lib/crypto/kimchi_backend/gadgets/range_check.ml